tests/test_utils.py

```diff
@@ -89,15 +91,15 @@ def test_sample_compression_1d():
     np.random.seed(0)
     N = 10000
     x_ = np.random.rand(N)
-    w_ = np.random.rand(N)
+    w_ = np.random.randn(N)
```

Comment:
Why the normally distributed weights, here? This will produce negative weights...

Reply:
I'll change it to the same log normal distribution used later on in the function.

Reply:
Why change it at all? I'd rather keep tests as is, unless there is a bug...

Reply:
I changed it to `w_ = np.random.lognormal(1, 0.5, N)` since `np.random.randn` will give you negative weights and these are unphysical, right? Would you ever draw a sample with a negative weight?

Reply:
But why change it to `randn` in the first place? Why not leave it as `rand`?

Reply:
Hmm I can't remember why I changed it... I have rolled it back to the `rand` function.